### PR TITLE
Update documentation about option --shouldSkipMarkup

### PR DESCRIPTION
**Pull request by qinezh**
@superyyrrzz @hellosnow @ansyral @vwxyzh @chenkennt #756

### CODE REVIEW

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 1]
```diff
@@ -77,6 +77,12 @@ Files can be combined using `,` as separator and *search pattern*.
 
 The default output folder is `_site/` folder if it is not specified in `docfx.json` under current directory.
 
+####2.2.1 Command option `--shouldSkipMarkup`
```
Comment (hellosnow) [inline Documentation/tutorial/docfx.exe_user_manual.md]: >2.2.1 [](start = 4, length = 5)

2.2.2?

**Comment by hellosnow** [comment]
:shipit:

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 2]
```diff
@@ -77,6 +77,12 @@ Files can be combined using `,` as separator and *search pattern*.
...
+
+If adding option `--shouldSkipMarkup` in metadata command, it means that DocFx would not render triple-slash-comments in source code as markdown.
```
Comment (superyyrrzz) [inline Documentation/tutorial/docfx.exe_user_manual.md]: >x [](start = 77, length = 1)

X

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 3]
```diff
@@ -144,6 +150,15 @@ Top level `docfx.json` structure is key-value pair. `key` is the name of the sub
...
 `Metadata` section defines an array of source projects and their output folder. Each item has `src` and `dest` property. `src` defines the source projects to have metadata generated, which is in `File Mapping Format`. Detailed syntax is described in **4. Supported `name-files` File Mapping Format** below. `dest` defines the output folder of the generated metadata files.
 
+Key                      | Description
+-------------------------|-----------------------------
+src                      | Defines the source projects to have metadata generated, which is in `File Mapping Format`.
+dest                     | Defines the output folder of the generated metadata files.
+force                    | If set to true, it would disable incremental build.
+shouldSkipMarkup         | If set to true, DocFX would not render triple-slash-comments in source code as markdown.
+filter                   | Defines the filter configuration file, please go to [How to filter out unwanted apis attributes](./howto_filter_out_unwanted_apis_attributes.md) for more details.
+useCompatibilityFileName | If set to true, DocFx would keep ``` in comment id other than replace it with `-`.
```
Comment (superyyrrzz) [inline Documentation/tutorial/docfx.exe_user_manual.md]: >x [](start = 47, length = 1)

X

Comment (qinezh) [inline Documentation/tutorial/docfx.exe_user_manual.md]: It's not a markup rule. It used for better displayed URL for managed reference.

Comment (hellosnow) [inline Documentation/tutorial/docfx.exe_user_manual.md]: you could try 'dfmPreview' extension in vs code, the syntax could be verified quickly rendering by dfm.

---
In reply to: [85038964](https://github.com/dotnet/docfx/pull/822#discussion_r85038964) [](ancestors = 85038964)

**Comment by superyyrrzz** [comment]
:shipit:

**Pull request merged**